packages/apps/auth0-actions/src/login.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title>Sign In with Auth0</title>
```

Comment:
Should this not be something else? We shouldn't be making it obvious to the end user that this is an Auth0 platform, that sort of thing should be invisible.